### PR TITLE
Update README and Github links for kinetic

### PR DESCRIPTION
**Pull request by davetcoleman**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,14 +1,18 @@
 # MoveIt! Tutorials
 
-This repo is automatically built by the ROS build farm and its output is hosted here: http://docs.ros.org/indigo/api/moveit_tutorials/html/
+This repo is automatically built by the ROS build farm and its output is hosted here: http://docs.ros.org/kinetic/api/moveit_tutorials/html/
```
Comment (v4hn) [inline README.md]: Does the buildfarm generate documentation for packages that are not released for this distribution yet?
The tutorials depend on pr2_arm_kinematics that is not released for kinetic.

Comment (davetcoleman) [inline README.md]: We are currently not "building" this package like a normal ROS package, only running doc jobs on it. https://github.com/ros/rosdistro/pull/12976/files

Comment (davetcoleman) [inline README.md]: ...so yes, the buildfarm will generate documentation for kinetic

its in queue: http://build.ros.org/job/Kdoc__moveit_tutorials__ubuntu_xenial_amd64/

**Comment by davetcoleman** [comment]
There are lots of dead links on the moveit_tutorials page because this PR is not merged. Merging.

**Pull request merged**